backend/src/main/java/edu/alexu/cse/dripmeup/Controller/UserSessionController.java

```diff
@@ -0,0 +1,103 @@
+package edu.alexu.cse.dripmeup.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.alexu.cse.dripmeup.Entity.UserEntity;
+import edu.alexu.cse.dripmeup.Repository.UserRepository;
+import edu.alexu.cse.dripmeup.Service.UserService;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:8080")
+@RequestMapping("users")
+
+public class UserSessionController {
+
+    @Autowired
+    private UserService userService;
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping("/login/{email}_{password}")
+
+    public ResponseEntity<?> login(@PathVariable String email, @PathVariable String password) {
+
+        boolean isAuthenticated = userService.login(email, password);
+        if (isAuthenticated) {
+            UserEntity user = userRepository.findByEmail(email);
+            UserEntity response = new UserEntity();
```

Comment:
change response to Person object and send to front

Reply:
okay i will handle it